tmt-app/src/Components/Stepper.jsx

```diff
@@ -0,0 +1,96 @@
+import * as React from "react";
+import { useLocation } from "react-router-dom";
+import { CustomTheme } from "../Styling/CustomStyling";
+import HomeIcon from "../Images/icons/boba_progress/Home_Current.png";
+import SubjectsIcon from "../Images/icons/boba_progress/Subjects_Current.png";
+import SoftSkillsIcon from "../Images/icons/boba_progress/SoftSkills_Current.png";
+import OrderUpIcon from "../Images/icons/boba_progress/OrderUp_Current.png";
+import {
+  Stepper,
+  Step,
+  StepLabel,
+  StepConnector,
+  stepConnectorClasses,
+  styled,
+} from "@mui/material";
+
+const ColorlibConnector = styled(StepConnector)(({ theme }) => ({
```

Comment:
What does `Colorlib` mean or stand for in `ColorlibConnector` ?

Reply:
I renamed the components in the latest commit.